src/core/main.c

```diff
@@ -531,6 +531,17 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
 
                 (void) parse_path_argument(value, false, &arg_watchdog_device);
 
+        } else if (proc_cmdline_key_streq(key, "systemd.watchdog_timeout")) {
```

Comment:
needs documentation

Reply:
also, probably should be called: watchdog_sec=

Reply:
I will once I'll be sure we want to introduce this option ;)